src/widget/wtracktableview.cpp

```diff
@@ -1975,6 +1984,18 @@ void WTrackTableView::slotReloadCoverArt() {
     }
 }
 
+void WTrackTableView::slotSortingChanged(int headerSection, Qt::SortOrder order) {
+    double sortOrder = (order == Qt::AscendingOrder) ? 0.0 : 1.0;
```

Comment:
Qt::AscendingOrder/DescendingOrder are well-defined (integer) constants 0/1 that we could use directly.

Reply:
True.